__tests__/app.test.js

```diff
@@ -44,17 +45,45 @@ describe("get topics", () => {
       });
   });
 
-    test("should respond with correct values", () => {
-      return request(app)
-        .get("/api/topics")
-        .expect(200)
-        .then((response) => {
-          const topics = response.body.topics;
-          expect(topics[0]).toHaveProperty("slug", "mitch");
-          expect(topics[0]).toHaveProperty(
-            "description",
-            "The man, the Mitch, the legend"
-          );
-        });
-    });
+  test("should respond with correct values", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
+        const { topics } = response.body;
+        expect(topics[0]).toHaveProperty("slug", "mitch");
+        expect(topics[0]).toHaveProperty(
+          "description",
+          "The man, the Mitch, the legend"
+        );
+      });
+  });
+});
+
+describe("get api", () => {
+  let endpoints;
+  beforeAll(async () => {
+    endpoints = await getFileContents("endpoints.json");
+  });
+
+  test("should respond with an object", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        const { apiDetails } = response.body;
+        console.log(apiDetails);
+        expect(typeof apiDetails).toBe("object");
```

Comment:
This would also pass if apiDetails was an Array, so you may need to add Array.isArray === false